apps/desktop/desktop/src/main/resources/css/user.css

```
@@ -414,4 +414,15 @@
\ No newline at end of file
+}
+
+
+/*******************************************************************************
+ * Add to contacts list popup                                                  *
+ ******************************************************************************/
+
+.add-to-contacts-list-window .material-text-field-bg,
+.add-to-contacts-list-window .material-text-field-bg-hover,
+.add-to-contacts-list-window .material-text-field-bg-selected {
+    -fx-background-radius: 4 4 0 0;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)